File: Dddml.Wms.HttpServices.JavaClient/src/org/dddml/wms/domain/OrganizationStructureTypeStateEventId.java
package org.dddml.wms.domain;


public class OrganizationStructureTypeStateEventId
{
    private String id;

    public String getId()
    {
        return this.id;
    }

    public void setId(String id)
    {
        this.id = id;
    }

    private Long version;

    public Long getVersion()
    {
        return this.version;
    }

    public void setVersion(Long version)
    {
        this.version = version;
    }

    OrganizationStructureTypeStateEventId ()
    {
    }

    public OrganizationStructureTypeStateEventId (String id, Long version)
    {
        this.id = id;
        this.version = version;
    }

}

File: Dddml.Wms.AdminUI/ts/OrganizationStructureTypeStateEventId.ts
namespace Dddml.Wms.Domain {

	export class OrganizationStructureTypeStateEventId {

		private id: string;

		getId(): string {
			return this.id;
		}

		setId(id: string) {
			this.id = id;
		}

		private version: string;

		getVersion(): string {
			return this.version;
		}

		setVersion(version: string) {
			this.version = version;
		}

	}
}

